wes schemas update & autogen (WIP)

achave11-ucsc opened the pull request:
Deleting checked in code for `swagger-codegen ` and adding method for generating schemas from maven build.

achave11-ucsc committed:
Generating java classes to support WES during maven build.
Process automation.

achave11-ucsc committed:
Removed unused properties, cleaning up.

achave11-ucsc committed:
Updating test, skiping lincense check test

achave11-ucsc committed:
Included generated artifact with autogenerated wes-schemas in the pom.

achave11-ucsc committed:
Removing unused classes preparing for refactoring and updating classes.

coverbeck commented on .tidelift.yml:
```diff
@@ -1,29 +1,6 @@
     development:
       tests:
-        unlicensed: skip
+        unlicensed: skipls
```
What is this change for? I'm not familiar with `unlicensed`.

coverbeck commented:
Why are all the .md files being deleted? Were those being generated as well?

coverbeck commented on swagger-ga4gh-client/pom.xml:
```diff
@@ -7,150 +7,107 @@
+              <configOptions>
+                <sourceFolder>src/gen/java/main</sourceFolder>
+                <library>jersey2</library>
+                <dateLibrary>java8</dateLibrary>
```
You have java.version 7 above, but you are specifying a dateLibrary of java8.

I don't know what the options are, but it seems like a potential problem. Should at least be researched.

achave11-ucsc replied:
https://github.com/swagger-api/swagger-codegen/issues/5863
Suggest that is only a name for the dateLibrary but it does not necessarily support
SDK8 java classes.

achave11-ucsc committed:
skip checking for license.

This is a CI tool, that check dependencies and compatibilities. 
Because some artifacts are being generated locally, they must contain 
a license for them to be tidelift compliant. Turning of tidelift ci testing.

achave11-ucsc committed:
Change the datelibrary to joda. Unclear java driver for consonance. Taking
safe approach for now.

achave11-ucsc committed:
Merge branch 'update/WES-schemas' of https://github.com/achave11/consonance into update/WES-schemas
